### DISCUSSION TITLE
One time donation

### DISCUSSION THREAD

**Discussion by rickysarraf** [photoprism/photoprism#1249] [General]
Thank you for photoprism.

I was wondering if there's a possibility to make a one-time donation than the current recurring model. I would like to express my gratitude, also with a small monetary donation.

**Comment by alexisLefebvre** [comment]
https://github.com/photoprism/photoprism shows a link to https://www.paypal.com/paypalme/photoprism I think you can donate once from there.

**Reply by rickysarraf** [reply]
Thanks. That made me recover my old (dormant) PayPal account.

And it sucks to not be able to make an easy transaction.
>We are sorry, but we could not complete this payment, because it has been denied by the card issuer. Please retry with a different card or funding option to continue with the payment.

**Comment by lastzero** [comment]
You can also do a yearly membership on Patreon and then cancel it if you don't want it to be recurring.

**Reply by rickysarraf** [reply]
I just tried through Patreon and it succeeded. Thanks for your commitment to PhotoPrism.

**Comment by graciousgrey** [comment]
One time donations are now supported by Github Sponsors as well: https://github.com/sponsors/photoprism

**Reply by rickysarraf** [reply]
Perfect. I hope you did receive the previous one-time donation through Patreon. 🙏🏽
